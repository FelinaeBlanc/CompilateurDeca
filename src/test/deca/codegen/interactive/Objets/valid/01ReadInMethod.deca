class A {

    int getInt() {
        return readInt();
    }

    void displayInt() {
       int x = readInt();
       println(x);  
    }

    float getFloat(){
        return readFloat(); 
    }

    void displayFloat(){
        float y = readFloat(); 
        println(y);
        printlnx(y);
    }
    

}

{
    A a = new A() ;
    println(a.getInt());
    a.displayInt();
    println(a.getFloat());
    a.displayFloat(); 
}
