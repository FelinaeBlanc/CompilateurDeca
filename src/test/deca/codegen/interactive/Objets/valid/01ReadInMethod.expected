10
15
1.21500e+01
4.84249e+02
0x1.e43fap+8
